views/js/qtiCreator/widgets/component/minMax/minMax.js

```diff
@@ -492,11 +546,7 @@ define([
 
                         fieldControl.input.on('change', function () {
                             self.syncValues(field);
-
-                            if (!config.allowDecimal) {
-                                self.convertToNumber(field);
-                            }
-
+                            self.convertToNumber(field);
```

Comment:
Now I can set negative number for `min`, but not for `max`. 

Why? I think `max` should be allowed to be negative too. If user can set negative score for interaction choice, then both `min` and `max` should be allowed to be negative too, because they limit that choice-score.

PS: Maybe it would have been better to reuse same input component which is used for choice-score? And leave `minMax.js` untouched. There would be also more space, now if you enter negative/float number, user can't even see whole number because input field is too small. But up to you.

![Screenshot_23](https://github.com/user-attachments/assets/231025cb-1bdd-4deb-9e84-a9c1d692bee0)

Reply:
I did `max` value as 0 or positive value in purpose, because of acceptance criteria: [AUT-1389](https://oat-sa.atlassian.net/browse/AUT-1389)

AC:
  a. This includes setting the:
    i. min  to 0, negative, or positive double-precision numbers
    ii. max to 0, or positive double-precision numbers

Reply:
Oh, ok, I didn't notice  that AC was updated.